plugins/test/unit/plugins/distributors/yum/metadata/test_metadata.py

```diff
@@ -456,13 +410,13 @@ def test_updateinfo_get_repo_unit_nevra_return(self):
         ]}]
 
         context = UpdateinfoXMLFileContext(self.metadata_file_dir, set([unit1_nevra]))
-        repo_unit_nevra = context._get_repo_unit_nevra(erratum_unit)
+        repo_unit_nevra = context.get_repo_unit_nevra(erratum_unit)
 
         self.assertEqual(len(repo_unit_nevra), 1)
         self.assertTrue(unit1_nevra_dict in repo_unit_nevra)
         self.assertTrue(unit2_nevra_dict not in repo_unit_nevra)
 
-    def test_updateinfo_get_repo_unit_nevra_no_epoch(self):
+    def test_updateinfoget_repo_unit_nevra_no_epoch(self):
```

Comment:
Bad wording...

I think that this test name was incorrect before your changes to UpdateinfoXMLContextThingyWhateverItsCalled, but the rename to `get_repo_unit_nevra` makes the old test name match our test naming convention. The correct name in this PR for this test should probably be `test_updateinfo_get_repo_unit_nevra_no_epoch`